ReactAndroid/build.gradle

```diff
@@ -240,6 +240,7 @@ dependencies {
     compile 'com.squareup.okhttp:okhttp-ws:2.4.0'
     compile 'com.squareup.okio:okio:1.5.0'
     compile 'org.webkit:android-jsc:r174650'
+    compile "org.java-websocket:Java-WebSocket:1.3.0"
```

Comment:
Agree with @foghina's suggestion. If you find that Java-WebSocket works better though you could always publish your own module to npm instead though.